kitsune/products/jinja2/products/blocks/frequent_topics_block.html

```diff
@@ -0,0 +1,55 @@
+<div class="mzp-l-content sumo-page-section--inner">
```

Comment:
this needs indentation. 2 spaces for html

Reply:
I'm assuming you mean the whole block, so I did that.